src/pages/blog/2020-airship-tc-election.md

```diff
@@ -0,0 +1,48 @@
+---
+templateKey: blog-post
+title: Congratulations To The 2020-2021 Technical Committee
+author: Alexander Hughes
+date: 2020-06-22T12:00:00.000Z
+category: 
+  - id: category-A7xLnYrE1
+    label: Election
+---
+
+The Airship team is excited to announce the completion of its second [_Technical Committee election_](
+http://lists.airshipit.org/pipermail/airship-discuss/2020-June/001005.html). The [_Technical Committee_](
+https://opendev.org/airship/governance#user-content-technical-committee), one of the two governing bodies for Airship,
+is responsible for ensuring Airship projects adhere to the project's core principles, promote standardization, and
```

Comment:
Would be nice to be able to link to the project's core pinciples here.

Reply:
Agreed, let's take this to the next WC meeting who is responsible for defining the project core principles and get this added to governance.